From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rezoss <rezoss79@users.noreply.github.com>
Date: Sun, 5 Jul 2026 00:00:00 +0200
Subject: [PATCH] Advertise Now Nudge in-app revision

Expose the metadata key read by SamsungSmartSuggestions when it checks
whether Samsung Messages supports in-app Now Nudge.

---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -282,6 +282,7 @@
         <uses-library android:name="textmining" android:required="false"/>
         <uses-native-library android:name="libagifencoder.quram.so" android:required="true"/>
         <meta-data android:name="oneUiVersion" android:value="80500"/>
+        <meta-data android:name="nownudge.inappnudge.revision" android:value="1"/>
         <meta-data android:name="theming-meta" android:value="meta_013_messages"/>
         <meta-data android:name="com.google.android.gms.car.application" android:resource="@xml/auto_message"/>
         <meta-data android:name="androidx.car.app.minCarApiLevel" android:value="7"/>
-- 
2.51.0

